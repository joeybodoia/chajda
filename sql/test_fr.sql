CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."');
