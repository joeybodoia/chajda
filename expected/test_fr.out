CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                   spacy_lemmatize                                    
--------------------------------------------------------------------------------------
   exemple:5 phrase:7 test:9 mets:12 google:14 translate:15 générer:17 cas:19 test:21
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                       spacy_lemmatize                        
--------------------------------------------------------------
   exemple phrase test mets google translate générer cas test
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                      spacy_lemmatize                                                                       
------------------------------------------------------------------------------------------------------------------------------------------------------------
   ceci:2 est:3 mon:4 exemple:5 de:6 phrase:7 de:8 test:9 que:10 je:11 mets:12 dans:13 google:14 translate:15 pour:16 générer:17 un:18 cas:19 de:20 test:21
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                            spacy_lemmatize                                             
--------------------------------------------------------------------------------------------------------
   ceci est mon exemple de phrase de test que je mets dans google translate pour générer un cas de test
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   ":2 exemple:6 phrase:8 test:10 mets:13 google:15 translate:16 générer:18 cas:20 test:22 .:23 ":24
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                          spacy_lemmatize                           
--------------------------------------------------------------------
   " exemple phrase test mets google translate générer cas test . "
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                              spacy_lemmatize                                                                              
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 ceci:3 est:4 mon:5 exemple:6 de:7 phrase:8 de:9 test:10 que:11 je:12 mets:13 dans:14 google:15 translate:16 pour:17 générer:18 un:19 cas:20 de:21 test:22 .:23 ":24
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                               spacy_lemmatize                                                
--------------------------------------------------------------------------------------------------------------
   " ceci est mon exemple de phrase de test que je mets dans google translate pour générer un cas de test . "
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                   spacy_lemmatize                                    
--------------------------------------------------------------------------------------
   exemple:5 phrase:7 test:9 mets:12 Google:14 Translate:15 générer:17 cas:19 test:21
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                       spacy_lemmatize                        
--------------------------------------------------------------
   exemple phrase test mets Google Translate générer cas test
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                      spacy_lemmatize                                                                       
------------------------------------------------------------------------------------------------------------------------------------------------------------
   Ceci:2 est:3 mon:4 exemple:5 de:6 phrase:7 de:8 test:9 que:10 je:11 mets:12 dans:13 Google:14 Translate:15 pour:16 générer:17 un:18 cas:19 de:20 test:21
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                            spacy_lemmatize                                             
--------------------------------------------------------------------------------------------------------
   Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer un cas de test
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   ":2 exemple:6 phrase:8 test:10 mets:13 Google:15 Translate:16 générer:18 cas:20 test:22 .:23 ":24
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                          spacy_lemmatize                           
--------------------------------------------------------------------
   " exemple phrase test mets Google Translate générer cas test . "
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                              spacy_lemmatize                                                                              
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 Ceci:3 est:4 mon:5 exemple:6 de:7 phrase:8 de:9 test:10 que:11 je:12 mets:13 dans:14 Google:15 Translate:16 pour:17 générer:18 un:19 cas:20 de:21 test:22 .:23 ":24
(1 row)

SELECT spacy_lemmatize('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                               spacy_lemmatize                                                
--------------------------------------------------------------------------------------------------------------
   " Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer un cas de test . "
(1 row)

SELECT spacy_tsvector('fr',' "Ceci est mon exemple de phrase de test que je mets dans Google Translate pour générer des cas de test."');
                                        spacy_tsvector                                         
-----------------------------------------------------------------------------------------------
 'cas':19 'exemple':5 'google':14 'générer':17 'mets':12 'phrase':7 'test':9,21 'translate':15
(1 row)

